pkg/cmd/cli/backup/create.go

```diff
@@ -138,12 +143,77 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		return err
 	}
 
+	var sharedInformerFactory informers.SharedInformerFactory
+	var updates chan *api.Backup
+	if o.Wait {
+		stop := make(chan struct{})
+		defer close(stop)
+
+		updates = make(chan *api.Backup)
+
+		sharedInformerFactory = informers.NewFilteredSharedInformerFactory(arkClient, 0, f.Namespace(), nil)
```

Comment:
I don't think we really need an informer factory, since we know we only need a single informer for one client - https://github.com/heptio/ark/blob/c8989231eb468f1ba9bdc531668fca8e2580c0ba/pkg/generated/informers/externalversions/ark/v1/backup.go#L54 might require fewer resources.